build/mech_converter.py

```diff
@@ -1,3 +1,4 @@
+-------------------------------------------------------------------------
```

Comment:
Needs a `#` to mark this line as a comment, not a command :)

Reply:
I'm on it :)